web/templates/recipient_row.hbs

```diff
@@ -40,9 +40,7 @@
 
             {{! exterior links (e.g. to a trac ticket) }}
             {{#each topic_links}}
-                <a href="{{this.url}}" target="_blank" rel="noopener noreferrer" class="external-topic-link recipient-bar-control no-underline">
-                    <i class="fa fa-external-link-square recipient_bar_icon" data-tippy-content="Open {{this.text}}" aria-label="{{t 'External link' }}"></i>
-                </a>
+                {{> components/icon_button icon="external-link" intent="neutral" custom_classes="external-topic-link recipient-bar-control recipient-bar-control-icon" data-url=this.url data-tippy-content=this.text aria-label="{{t 'External link' }}" }}
```

Comment:
No, we should keep an `a` tag -- the fact that the browser displays a preview of the URL is important to usability. I think you should be able to just have an `a` tag around the icon button?

Reply:
Makes sense, and just wrapping the button with an `a` tag did the job. Updated.

Reply:
```suggestion
                <a href="{{this.url}}" target="_blank" rel="noopener noreferrer" class="recipient-bar-control no-underline">
                    {{> components/icon_button icon="external-link" intent="neutral" custom_classes="external-topic-link recipient-bar-control-icon" data-tippy-content=this.text aria-label="{{t 'External link' }}" }}
                </a>
```

Reply:
Referencing the recent development over at https://github.com/zulip/zulip/pull/34777#discussion_r2184119483.